.gitignore

```diff
@@ -26,3 +26,7 @@ goimports-check
 vet
 whitespace
 whitespace-check
+
+# bumper go exec file
```

Comment:
Can we build it in the _out or build_output directory instead?

Reply:
I was thinking of ultimately running the auto-bump via the `Makefile` and that runs on the primary folder.
So building it on the _out folder only so that the bumper exec file will be ignored might look awkward if you stick to my makefile paradigm.. 
What do you think?

Reply:
`go run` generates the binary in the root folder?

Reply:
since our script is comprised out of more than 1 file, we need to build it first. this creates the bumper exec file

Reply:
Ok, but then you can specify the destination for the file. Just like we do it with all the other binaries https://github.com/kubevirt/cluster-network-addons-operator/blob/master/Makefile#L54

Reply:
ack. ok so this PR is not needed then..